include/common.sh

```diff
@@ -31,12 +31,14 @@ set +x
 export DOCKER_USERNAME="${DOCKER_USERNAME:-}"
 export DOCKER_PASSWORD="${DOCKER_PASSWORD:-}"
 export CLUSTER_PASSWORD="${CLUSTER_PASSWORD:-password}"
+set -x
```

Comment:
I would rather avoid to be verbose here by default, [debug_mode](https://github.com/SUSE/catapult/blob/master/include/func.sh#L150) is called [here](https://github.com/SUSE/catapult/blob/master/include/common.sh#L6) so it will get verbose if `DEBUG_MODE` is set

Reply:
Argh, true, thanks!
The problem though is that after the `set +x` for `DOCKER_*` creds, we want to reenable if `DEBUG_MODE=true` or we don't list the rest. I will substitute it with a call to `debug_mode`.